providers/tests/system/openlineage/transport/variable.py

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from openlineage.client.serde import Serde
+from openlineage.client.transport import Transport, register_transport
+
+from airflow.models.variable import Variable
+from airflow.utils.log.logging_mixin import LoggingMixin
+
+if TYPE_CHECKING:
+    from openlineage.client.client import Event
+
+
+@register_transport
+class VariableTransport(Transport, LoggingMixin):
+    kind = "variable"
+    """
+    This transport sends OpenLineage events to Variables.
+
+    Key schema is <DAG_ID>.<TASK_ID>.event.<EVENT_TYPE>.
+    It's made to be used in system tests, stored data read by OpenLineageTestOperator.
+    """
```

Comment:
```suggestion
    """
    This transport sends OpenLineage events to Variables.

    Key schema is <DAG_ID>.<TASK_ID>.event.<EVENT_TYPE>.
    It's made to be used in system tests, stored data read by OpenLineageTestOperator.
    """
    
    kind = "variable"
```

It looks like the docstring is for the class instead? if that's the case, probably reorder it this way would be better

Reply:
Fixed.